rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -327,6 +327,16 @@ def test_exec(config, ssh_con):
                                 else:
                                     raise AssertionError("Abort multipart failed")
 
+                        elif config.test_ops.get("upload_type") == "failed_upload":
+                            log.info("Failed object upload")
+                            reusable.failed_upload_object(
+                                s3_object_name,
+                                bucket,
+                                TEST_DATA_PATH,
+                                config,
+                                each_user,
+                            )
```

Comment:
@TejasC88 , can we add a bypass to upload failure as upload from read_only site is expected  to fail.

Reply:
@mkasturi18 I have added a if-else block , is that what's needed here?